src/test/java/net/snowflake/ingest/TestUtils.java

```diff
@@ -199,7 +204,6 @@ public static Connection getConnection() throws Exception {
     Properties props = new Properties();
     props.put("user", user);
     props.put("account", account);
-    props.put("ssl", ssl);
```

Comment:
why remove this? Similarly for above ssl?

Reply:
I don't think this is being used, does it?

Reply:
if we dont pass ssl in properties, the Connection wont be secure.